Sources/Starknet/Providers/StarknetProvider/StarknetProvider.swift

```diff
@@ -4,7 +4,7 @@ public enum StarknetProviderError: Error {
     case networkProviderError
     case unknownError
     case jsonRpcError(Int, String, String?)
-    case emptyBatchRequestError
+    case emptyRequestsError
```

Comment:
note: Seems to me that this error name is more suitable once `StarknetBatchRequest` has been removed.

Reply:
Hmm, I think the previous name was ok and indicated underlying use of batch request. Otherwise we might consider changing it to something like `noRequestsError`, as current name is a bit confusing